test/safe/branch.test.ts

```diff
@@ -163,8 +161,6 @@ describe("Branch Webhook", () => {
 	describe("Create Branch (with disabled FF - delete this test with FF cleanup)", () => {
 		it("should update Jira issue with link to a branch on GitHub", async () => {
 
-			mockGitHubAuthRequest();
```

Comment:
So, we need to have it here too, otherwise if this test gets executed first it will fail